docs/packaging.rst

```diff
@@ -0,0 +1,57 @@
+====================
+Packaging guidelines
+====================
+
+Thank you very much for packaging vdirsyncer! The following guidelines should
+help you to avoid some common pitfalls.
+
+While they are called guidelines and therefore theoretically not mandatory, if
+you consider going a different direction, please first open an issue or contact
+me otherwise instead of just going ahead. These guidelines exist for my own
+convenience too.
+
+Obtaining the source code
+=========================
+
+The main distribution channel is `PyPI
+<https://pypi.python.org/pypi/vdirsyncer>`_, and source tarballs can be
+obtained there. Do not use the ones from GitHub: Their tarballs contain useless
+junk and are more of a distraction than anything else.
+
+I give each release a tag in the git repo. If you want to get notified of new
+releases, `GitHub's feed
+<https://github.com/untitaker/vdirsyncer/releases.atom>`_ is a good way.
+
+Dependency versions
+===================
+
+It is strongly discouraged to package vdirsyncer as a Python 2 application.
+Future releases will only work on Python 3.3 and newer versions.
+
+As with most Python packages, ``setup.py`` denotes the runtime dependencies of
+vdirsyncer. It also contains lower-bound versions of each dependency. Older
+versions may appear to work (since e.g. all tests pass), but they really don't.
```

Comment:
is it possible for you to write a unit test that fails if a lower bound is not satisfied? You would get less bug reports I guess.